statics/src/test/java/integration/SelenideMethodsTest.java

```diff
@@ -564,6 +565,20 @@ void canExecuteCustomCommand() {
     assertThat(mirrorText).startsWith("custom value");
   }
 
+  @Test
+  void canExecuteJavascript() {
+    long value = (Long) Selenide.executeJavaScript("return 10;");
+    assertThat(value).isEqualTo(10);
+  }
+
+  @Test
+  void canExecuteAsyncJavascript() {
+    long value = (Long) Selenide.executeAsyncJavaScript(
+      "var callback = arguments[0]; setTimeout(function() { callback(10); }, 50);"
```

Comment:
The proper way is `arguments[arguments.length - 1]` instead of `arguments[0]`.

Reply:
done in my next commit